perfkitbenchmarker/publisher.py

```diff
@@ -134,13 +135,22 @@ def AddMetadata(self, metadata, benchmark_spec):
     metadata['zones'] = ','.join(benchmark_spec.zones)
     metadata['machine_type'] = benchmark_spec.machine_type
     metadata['image'] = benchmark_spec.image
+
+    # Scratch disk is not defined when a benchmark config is provided.
+    if getattr(benchmark_spec, 'scratch_disk', None):
+      metadata.update(scratch_disk_type=benchmark_spec.scratch_disk_type,
```

Comment:
The number of disks striped together is something else that should be added to the metadata since it will definitely affect performance.